src/components/RecentSearches.js

```diff
@@ -1,16 +1,17 @@
 import React, { Component } from 'react';
 
-const RecentSearches = ({ history, onClick }) =>
+const RecentSearches = ({ history, onClick, clearHistory }) =>
   (
-    <div className='tc dib pa2 br3 ma0 bw2 shadow-2 bg-light-blue'>
-      <h3>Last 5 Searches</h3>
+    <div className='tc pa1 br3 ma0 bw2 shadow-2 bg-light-blue'>
+      <h3 className='dib ph3'>{history.length===0 ? 'Search to start a History' : 'Search History'}</h3>
```

Comment:
Usually I put the 'empty state' message in place of the data (below where you are doing `history.map`), not the header.

Reply:
Just made this change, and I like it better. thanks. done.